src/App.js

```diff
@@ -1,16 +1,27 @@
-//git pull로 다시 받아와서 재실행
+//css 추가
+import styles from './Button.module.css';
+import style from './Reset.module.css';
 import { useState } from 'react';
+import { useEffect } from 'react';
```

Comment:
```suggestion
import { useState, useEffect } from 'react';
```
둘 다 `react` 에서 꺼내 주고 있으므로 이렇게 써도 될 것 같아.